README.md

```diff
@@ -1,16 +1,83 @@
 # Tech Tests
 
-This repository works a question bank that volunteers can use for conducting mock interviews.
+This repository works as a question bank that volunteers can use for conducting mock interviews.
 
-## How to use it
+## How do the Tests Comprise a Mock Interview?
+
+For mock interviews to be effective, you have to expose the candidates to as many potential assessment methods as possible, so that they are well prepared for the real thing.
+
+Part of this will involve a well-conducted technical challenge in order to assess the following characteristics:
+
+* A candidate's approach to finding a solution to the problem
+* A candidate's ability to design a solution to the problem and describe the design
+* A candidate's approach to test-driven design and development
+* The quality of the implemented solution itself
+
+A mistake often made is just focussing on the last point, but employers of good software engineers will look at all of the above.
+
+When assessing a candidate's attempt at one of these tests, be sure to assess and give feedback on all of the above. There will be other questions and exercises beyond these tech tests too. **The aim of this is to ensure all technical tests are conducted consistently during mock interviews.**
+## How to use these Tests
 
 1. Pick one of the top-level folders for the category that best describes your question type:
     - data-structures-and-algorithms - Usually language agnostic. Focus more on correct use of logic, algorithms and data structures
     - homework-assignment - An assignment to be given to the candidate to be completed offline
-    - live-assignment - A task for the candidate to complete in a live coding session
-2. Create a new folder for your question with a descriptive name inside the category folder
-3. Inside your question folder, create a folder named `exercise`. A zip version of this folder will be given to the interviewee
-4. Inside the `exercise` folder, add a `README.md` file. Ideally the file should contain:
+    - live-assignment - A task for the candidate to complete in a live coding session or mock interview
+1. Within the folder chosen in 1. there will be some question folders, each of which contains a different technical test.
```

Comment:
```suggestion
2. Within the folder chosen in 1. there will be some question folders, each of which contains a different technical test.
```

Maybe this got reverted during the rebase. Can you please update all the numbered lists?